source/clear-linux/tutorials/web-server-install/web-server-install.rst

```diff
@@ -1,479 +1,423 @@
+   "This web server is operational from its new location."
 
-.. code-block:: console
+#. Stop and then restart ``httpd.service``.
 
-   sudo systemctl stop httpd.service
-   sudo systemctl start httpd.service
+   .. code-block:: bash
 
+      sudo systemctl stop httpd.service
+      sudo systemctl start httpd.service
 
-Now when you go to http://localhost you should see your new screen.
 
-To continue, we must change the configuration back to the default
-:file:`/var/www/html` location. To do this, edit the 
-:file:`/etc/httpd/conf.d/httpd.conf` file again and replace any instance of
-/var/www/tutorial with /var/www/html.
+#. Go to http://localhost to view the new screen.
 
-Then, stop and then restart the ``httpd.service``.
+#. Change the configuration back to the default :file:`/var/www/html` location. To do this, edit the :file:`/etc/httpd/conf.d/httpd.conf` file again and replace any instance of /var/www/tutorial with /var/www/html.
 
-.. code-block:: console
+#. Stop and then restart ``httpd.service``.
 
-   sudo systemctl stop httpd.service
-   sudo systemctl start httpd.service
+   .. code-block:: bash
 
-On http://localhost, you should see the default screen again.
+      sudo systemctl stop httpd.service
+      sudo systemctl start httpd.service
 
-Optionally, remove the /var/www/tutorial directory previously created.
+#. Go to http://localhost and verify that you can see the default screen again.
 
-.. code-block:: console
+   Optionally, remove the /var/www/tutorial directory you previously created.
 
-   sudo rm /var/www/tutorial/index.html
-   sudo rmdir /var/www/tutorial
+   .. code-block:: bash
 
-Installing PHP
+      sudo rm /var/www/tutorial/index.html
+      sudo rmdir /var/www/tutorial
+
+Install PHP
 **************
 
-With Apache installed, you can display static web pages. However, enabling
-PHP allows dynamic webpages to be generated and displayed. To add this
-functionality to your web server we need to install PHP on your system.
+An Apache installation allows you to display static web pages. Enabling PHP allows you to generate and display dynamic web pages. To add this functionality to your web server, install PHP on your system.
 
-To get the php components, enter the following command:
+#. To get the php components, enter the following command:
 
-.. code-block:: console
+   .. code-block:: bash
 
-   sudo swupd bundle-add php-basic
+      sudo swupd bundle-add php-basic
 
-To enable PHP, enter the following commands:
+#. To enable PHP, enter the following commands:
 
-.. code-block:: console
+   .. code-block:: bash
 
-   sudo systemctl enable php-fpm.service
-   sudo systemctl start php-fpm.service
-   sudo systemctl restart httpd.service
+      sudo systemctl enable php-fpm.service
+      sudo systemctl start php-fpm.service
+      sudo systemctl restart httpd.service
 
-After restarting the Apache service, we can test our PHP installation.
+After restarting the Apache service, test your PHP installation.
```

Comment:
Please indent Line 172 so that it's justified with the line above it. Doing this will make the steps in this section continuous. This is a subtlety of Sphinx/ReST.  I recommend using CTRL+ALT+O to view this change before pushing.